src/rgw/rgw_op.cc

```diff
@@ -6488,14 +6498,27 @@ void RGWAbortMultipart::execute(optional_yield y)
   op_ret = -EINVAL;
   string upload_id;
   upload_id = s->info.args.get("uploadId");
-  rgw_obj meta_obj;
+  std::unique_ptr<rgw::sal::Object> meta_obj;
   std::unique_ptr<rgw::sal::MultipartUpload> upload;
 
   if (upload_id.empty() || rgw::sal::Object::empty(s->object.get()))
     return;
 
   upload = s->bucket->get_multipart_upload(s->object->get_name(), upload_id);
   RGWObjectCtx *obj_ctx = static_cast<RGWObjectCtx *>(s->obj_ctx);
+
+  meta_obj = upload->get_meta_obj();
+  meta_obj->set_in_extra_data(true);
+  op_ret = meta_obj->get_obj_attrs(obj_ctx, s->yield, this);
+  if (op_ret < 0) {
+    ldpp_dout(this, 0) << "ERROR: failed to get obj attrs, obj=" << meta_obj
+		     << " ret=" << op_ret << dendl;
+    return;
```

Comment:
we probably don't want to return here. if we cannot fetch the attribute, we should just not set the trace.

Reply:
this is what caused the failure in tests. thanks.

Reply:
please also change from "ERROR" to warning and change the debug level to something different than zero

Reply:
I deleted the logging at all.  I thought it's unnecessary to log a warning/error in case of failure in tracing (although this warning is ceph-related ).